class/actions_subtotal.class.php

```diff
@@ -257,12 +259,12 @@ function formObjectOptions($parameters, &$object, &$action, $hookmanager)
 
 	function printNewFormat(&$object, &$conf, &$langs, $idvar)
 	{
-		if (empty($conf->global->SUBTOTAL_ALLOW_ADD_BLOCK)) return false;
+		if (!getDolGlobalString('SUBTOTAL_ALLOW_ADD_BLOCK')) return false;
 
 		$jsData = array(
 			'conf' => array(
-				'SUBTOTAL_USE_NEW_FORMAT' => !empty($conf->global->SUBTOTAL_USE_NEW_FORMAT),
-				'MAIN_VIEW_LINE_NUMBER' => !empty($conf->global->MAIN_VIEW_LINE_NUMBER),
+				'SUBTOTAL_USE_NEW_FORMAT' => getDolGlobalString('SUBTOTAL_USE_NEW_FORMAT'),
+				'MAIN_VIEW_LINE_NUMBER' => getDolGlobalString('MAIN_VIEW_LINE_NUMBER'),
```

Comment:
Attention cette variable sert à transmettre les info vers js 

```suggestion
				'SUBTOTAL_USE_NEW_FORMAT' => getDolGlobalInt('SUBTOTAL_USE_NEW_FORMAT'),
				'MAIN_VIEW_LINE_NUMBER' => getDolGlobalInt('MAIN_VIEW_LINE_NUMBER'),
```